src/layout/main.tsx

```diff
@@ -1,32 +1,38 @@
 import * as React from 'react';
+import { MemberProps } from '../templates/member-profile.js';
 
-type Props = {
+type MainProps = {
+  name: string;
+  member: MemberProps[];
   children: JSX.Element;
 }
 
-export function MainLayout(props: Props) {
+export function MainLayout(props: MainProps) {
 
-  const avatar = `
-+----------+
-|  ___   o |
-| /_|_\\    |
-|~~~|~~~~~~|
-|~~~|~~~~~~|
-|   | \\__  |
-+----------+
-`;
+  return (
+    <html>
+      <head>
+        <title> Members of TorontoJS </title>
+        <link href="/assets/style.css" rel="stylesheet" type="text/css" />
+      </head>
+      <body>
+        <figure role="img" aria-label="ASCII art of a beach scene">
+          <pre>
+            {`
+          +----------+
+          |  ___   o |
+          | /_|_\\    |
+          |~~~|~~~~~~|
+          |~~~|~~~~~~|
+          |   | \\__  |
+          +----------+
+          `}
```

Comment:
This is now broken prboably =(

Reply:
it should look like 
![Screenshot 2024-05-18 at 4 47 00 PM](https://github.com/torontojs/members/assets/6406037/007a626f-4316-4276-97d7-2135fce9c4a9)

Reply:
So my original idea was to build an ascii-only profile so we could also rehost this on the Gopher protocol, so the avatar should be part of the user type probably!

Reply:
But anyway this is yours now so go right ahead!

Reply:
Was it supposed to animate or...?